File: gui/device_selector.py
from PyQt5.QtWidgets import QGroupBox

from ui_device_selector import Ui_PwIODeviceSelector

class PwIODeviceSelector(QGroupBox, Ui_PwIODeviceSelector):
	def __init__(self, parent):
		QGroupBox.__init__(self, parent)
		self.setupUi(self)

File: gui/device_selector.cpp
#include "device_selector.hpp"
#include "device_selector.moc"

#include <QtWidgets/QWidget>
#include <QtWidgets/QGroupBox>

PwIODeviceSelector::PwIODeviceSelector(QWidget *parent)
	: QGroupBox(parent)
{
	ui.setupUi(this);
}

PwIODeviceSelector::~PwIODeviceSelector() = default;
